pywren/pywren_ibm_cloud/wrenconfig.py

```diff
@@ -46,13 +47,17 @@ def load(config_filename):
     if res['ibm_cf']['api_key'] == '<CF_API_KEY>':
         raise Exception(
             "{} has CF API key as {} -- make sure you change the default CF API key".format(
-                config_filename, res['pywren']['api_key']))
-    
+                config_filename, res['ibm_cf']['api_key']))
+
     if res['ibm_cos']['endpoint'] == '<COS_API_ENDPOINT>':
         raise Exception(
             "{} has CF API endpoint as {} -- make sure you change the default COS API endpoint".format(
-                config_filename, res['pywren']['storage_container']))
-    
+                config_filename, res['ibm_cos']['endpoint']))
+    if res['ibm_cos']['api_key'] == '<COS_API_KEY>':
+        raise Exception(
+            "{} has CF API key as {} -- make sure you change the default COS API key".format(
+                config_filename, res['ibm_cos']['api_key']))
+
     return res
```

Comment:
@JosepSampe why do we need this? api_key is not mandatory...it can be api_key or the pair access_key and secret_key. But this new line throw an exception if api key is not present. This is new addition, I tried it and got exception "KeyError: 'api_key'"

Reply:
I fixed the issue.